.travis.yml

```diff
@@ -14,11 +14,14 @@ env:
     - PATH=$HOME/cached-deps:$PATH
   matrix:
     - BUCKET=MISC
+    - BUCKET=AUTH1
+    - BUCKET=AUTH2
     # If you want to update the number of PPS workers, you'll also need to update
     # the value in ./etc/build/PPS_BUILD_BUCKET_COUNT
```

Comment:
is this still the case?

Reply:
Ah, good catch. I updated the comment, and also set the total number of PPS and auth buckets in env vars in the same file (right above here) so it's easier to update